### DISCUSSION TITLE
wyoming-satellite setup.

### DISCUSSION THREAD

**Discussion by Jagsta2121** [pgale/lineage_15.1_Installation_TSV#12] [Q&A]
does anybody have wyoming-satellite working on this? it is possible?
Johnathan

**Comment by pgale** [comment]
Ask on the View Assist Discord server. Lots of work going on around this. 

https://discord.gg/nnNpK2cg

Not sure anyone has local wake word running on the device but by using rtpmic app and streaming to an HA integration, streamassist, this works really well. 

Also have a look at the View Assist docs and loads of videos on how to set it up. View Assist is like an Alexa View experience. 

https://github.com/dinki/View-Assist

**Comment by Manichee** [comment]
I am using the scripts provided [here](https://github.com/pantherale0/wyoming-satellite-termux)

**Reply by contagon** [reply]
Just tried this, 100 times better than stream assist.

Has a few small bugs for setup (mine didn't enable auto start), but once I got it setup it's been great. It handles the timing of responses a lot better than stream assist.
